htdocs/takepos/takepos.php

```diff
@@ -450,7 +450,17 @@ function MoreActions(totalactions){
 
 <?php
 // TakePOS setup check
-if (empty($conf->global->CASHDESK_ID_THIRDPARTY) or empty($conf->global->CASHDESK_ID_BANKACCOUNT_CASH) or empty($conf->global->CASHDESK_ID_BANKACCOUNT_CB)) {
+// TODO
+$sql="SELECT code,libelle FROM ".MAIN_DB_PREFIX."c_paiement WHERE active=1 ORDER BY libelle";
+$resql = $db->query($sql);
```

Comment:
@dbeniamine 
for compatibility with multicompany :

```
$sql = "SELECT code, libelle FROM ".MAIN_DB_PREFIX."c_paiement";
$sql.= " WHERE entity IN (".getEntity('c_paiement').")";
$sql.= " AND active = 1";
$sql.= " ORDER BY libelle";
```

Reply:
done in 32303bbe4827cba26e8b57b5d17844766a1fc2c9